src/components/appLayout.tsx

```diff
@@ -80,7 +80,9 @@ class AppLayout extends Component<AppLayoutProps, any> {
         <Menu.Item key="wallet">
           <Link to="/wallet">Wallet</Link>
         </Menu.Item>
-        <Menu.Item key="faucet">
+        <Menu.Item key="faucet" style={{ 
```

Comment:
Why not
```jsx
{CONFIG.tokenFaucet && (
  <Menu.Item key="faucet">
    <Link to="/faucet">Get tokens</Link>
  </Menu.Item>
)}
```
?

Reply:
Good point and I have no answer. For some reason I made the same for `/slots` before and now just mindlessly and shamefully copied that. Fixed